includes/class-wp-job-manager-usage-tracking-data.php

```diff
@@ -1,17 +1,20 @@
 <?php
 /**
  * Usage tracking data
+ *
+ * @package Usage Tracking
  **/
+
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 
-/**
- * Supplies the usage tracking data for logging.
- *
- * @package Usage Tracking
- * @since 1.30.0
- */
+	/**
```

Comment:
Is there a reason to have this indented?

Reply:
Hmm...the linter was complaining about this earlier but seems to be fine when I outdent it again as per https://github.com/Automattic/WP-Job-Manager/pull/1331/commits/dfb0fec392ff30223cdd1467b11118c0378daeb3. 🤷‍♀️